js/controller.js

```diff
@@ -353,6 +379,11 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         this.enableFullScreen();
         this.updateAspectRatio();
       }
+      if (this.videoVr) {
+        if (window.DeviceOrientationEvent) {
+          window.addEventListener('deviceorientation', this.handleVrMobileOrientation.bind(this), false);
```

Comment:
Note that `bind` is returning a new function here, so calling `removeEventListener` with `this.handleVrMobileOrientation` will not remove this listener.

Reply:
\+